tests/lib/snaps/browser-support-consumer/bin/cmd

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+command="$1"
+shift
+
+"$command" "$@"
```

Comment:
Yeap! :) I'll update the generic-consumer snap too, a PR is coming with some refactors to use it, i'll include the changes there